services/api/internal/db/repository/README.md

```
@@ -70,6 +70,60 @@
+    GetAllUsers(ctx context.Context) ([]*User, error)
+    UpdateUser(ctx context.Context, id, email, name string) (*User, error)
+    DeleteUser(ctx context.Context, id string) error
+}
+
+// Implementation using sqlc
+type userRepository struct {
+    queries *sqlc.Queries
+}
+```
+
+#### Usage:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)